test/promql/tests/data_plane/release_service_test.yaml

```diff
@@ -57,3 +57,24 @@ tests:
               alert_team_handle: <!subteam^S03SVBS426R>
               runbook_url: https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/release-service/release-validation-latency.md?ref_type=heads
               team: release
+
+  - interval: 1m
+    input_series:
+      - series: 'kube_pod_container_status_ready{namespace="release-service", container="manager"}'
```

Comment:
We have `sum` on the query, but we only have a single data series on the tests, so we don't test summing.

Reply:
indeed.

Reply:
added samples from different clusters.